examples/ui-demo/src/components/configuration/Authentication.tsx

```diff
@@ -86,43 +69,43 @@ export const Authentication = ({ className }: { className?: string }) => {
           <AuthMethod
             icon={<MailIcon />}
             name="Email"
-            active={config.auth.showEmail}
+            active={auth.showEmail}
             disabled
           />
           <AuthMethod
             icon={<SocialIcon />}
             name="Social"
             iconClassName="mt-[2px] self-start"
             details={
-              config.auth.showOAuth && (
-                <div className="flex gap-x-3">
-                  <OAuthMethod
-                    active={config.auth.oAuthMethods.google}
-                    icon={<GoogleIcon />}
-                    onClick={setAddGoogleAuth}
-                  />
-                  <OAuthMethod
-                    active={config.auth.oAuthMethods.facebook}
-                    icon={<FacebookLogo />}
-                    onClick={setAddFacebookAuth}
+              <div
+                className={cn("flex gap-x-3", auth.showOAuth ? "" : "hidden")}
```

Comment:
(Very nit) can be written more simply as
```ts
cn("flex gap-x-3", !auth.showAuth && "hidden")
```
or
```ts
cn("flex gap-x-3", { hidden: !auth.showAuth })
```

Reply:
oh yea that's cleaner. I'll go with the second one